mautic/mautic
# Fix form issues with file type field

Fixes: #9946


| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.3
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
I've noticed two issue in forms. This PR fixed it.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create form with **file**, **email** and **file** and **Send form results** action
3. Disable  Save result? option for **file** and enable Public accessible link to download
4. Now send form
5. Without fix, you receive email with relative path to file. After apply the fix receive URL to download file
6. Then try clone form, and submit cloned form . Without fix you should see error in logs :

> Uncaught PHP Exception Doctrine\DBAL\Exception\InvalidFieldNameException: "An exception occurred while executing 'INSERT INTO

7. After fix and cloned form, you are able to submit cloned form without issue
